app/packages/app/src/components/catalog/__snapshots__/EntityPage.test.tsx.snap

```diff
@@ -1943,6 +1943,71 @@ exports[`[kind: component] [type: helm] /ci-cd Should render correctly when cicd
 ]
 `;
 
+exports[`[kind: component] [type: helm] /ci-cd Should render correctly when cicd is not available: content 2`] = `
```

Comment:
Not sure where this came from, when running the tests locally this shows up as an obselete snapshot which will cause the CI to error

Reply:
Strange, CI didnt fail but it should have according to the jest docs
https://jestjs.io/docs/snapshot-testing#are-snapshots-written-automatically-on-continuous-integration-ci-systems